pod/perldelta.pod

```diff
@@ -192,8 +192,8 @@ include any changes in L<perldiag> that reconcile it to the C<C> code.
 
 =head2 New Diagnostics
 
-XXX Newly added diagnostic messages go under here, separated into New Errors
-and New Warnings
+XXX Newly added diagnostic messages go under here, separated into L</New Errors>
+and L</New Warnings>
```

Comment:
Link to the sections, a practice done previously within the file with other sections too